app/services/katello/pulp/content_counts_calculator.rb

```diff
@@ -24,18 +24,18 @@ def calculate
           case
           when repo_type?(repo, 'deb')
             counts[:apt_repositories] += 1
-            counts[:deb_packages] += get_unit_count(repo, 'deb')
-          when repo_type?(repo, 'rpm')
+            counts[:deb_packages] += repo.debs.count
+          when repo_type?(repo, 'yum')
             counts[:yum_repositories] += 1
-            counts[:packages] += get_unit_count(repo, 'rpm')
-            counts[:package_groups] += get_unit_count(repo, 'package_group')
-            counts[:errata] += get_unit_count(repo, 'erratum')
+            counts[:packages] += repo.rpms.count
+            counts[:package_groups] += repo.package_groups.count
+            counts[:errata] += repo.errata.count
           when repo_type?(repo, 'docker')
             counts[:docker_repositories] += 1
-            counts[:docker_images] += get_unit_count(repo, 'docker_image')
+            counts[:docker_images] += repo.docker_manifests.count
           when repo_type?(repo, 'puppet')
             counts[:puppet_repositories] += 1
-            counts[:puppet_modules] += get_unit_count(repo, 'puppet_module')
+            counts[:puppet_modules] += repo.puppet_modules.count
```

Comment:
Puppet count won't work since we don't maintain puppet repo sync history. We could remove the puppet column from the page?